### DISCUSSION TITLE
Missing tools on desktop browser

### DISCUSSION THREAD

**Discussion by Tiritibambix** [LycheeOrg/Lychee#2405] [Q&A] [answered]
Tools like Fav, rotate, move or delete photo are missing. Tried different browers.

![Desktop](https://clip.steph.click/-tZiFCM7nPZ/scrPC.png)

I see them on mobile though

![Mobile](https://clip.steph.click/-vgef3C9HdH/IMG_20240424_194827.jpg)

**Comment by ildyria** [comment, marked as answer]
_Moved to the right place._

Now, to answer your question, have you tried mouse over the upper half of the image?

![image](https://github.com/LycheeOrg/Lychee/assets/627094/01ec59a4-17e3-4515-bfa9-a874b43c4cc9)

**Reply by Tiritibambix** [reply]
>Moved to the right place.

Thanks

>have you tried mouse over the upper half of the image?

I haven't before. It was not very intuitive and not very visible. To my eyes at least.
![dropdown](https://clip.steph.click/-GPDycDLpt2)

Also, this menu could have some amount of opacity in the background for readability
![add](https://clip.steph.click/-U96tr5Sy7d)

And I feel like something should appear in this big empty space. I might have misconfigured something. It is empty in each album
![empty](https://clip.steph.click/-B7XApgBBhV)

**Reply by ildyria** [reply]
The fact this is empty though is not normal, I will try to dig into that.

**Reply by ildyria** [reply]
> Also, this menu could have some amount of opacity in the background for readability

That is weird. I will dig into that.

**Reply by ildyria** [reply]
> Also, this menu could have some amount of opacity in the background for readability

Something is weird with your install. I cannot reproduce that one. O.o

**Reply by Tiritibambix** [reply]
I just tried a fresh install and I still have this problem. How should it display ?

**Reply by ildyria** [reply]
![image](https://github.com/LycheeOrg/Lychee/assets/627094/cf6f0142-8b82-4819-9450-894cd72e9c1b)
Like this.

**Reply by ildyria** [reply]
@tiritibambix what web-browser are you using ? 
What is your OS?

**Reply by Tiritibambix** [reply]
You got it. I'm using Librewolf (Firefox Fork). I tried with Firefox, Vivaldi and Ungoogled Chomium and it's working fine.
Empty space on top of albums is back to normal too. Good job on finding the issue.

But tools for photo are still no very visible nor very intuitive to find.

**Reply by ildyria** [reply]
> But tools for photo are still no very visible nor very intuitive to find.

it was a way to de-cluster the large number of icons in the top right.
I had bigger icons in mind before but the feedback I got is "it's too big/childish".
I also added shadow under it which makes them pop.
My guess is that your web client does not support gradient background and shadow well. :/

**Reply by Tiritibambix** [reply]
Thank you for investigating

**Comment by Tiritibambix** [comment]
Just wanted to chime in on this thread! I recently did a fresh install of Docker v4.9.0 regarding [this issue](https://github.com/LycheeOrg/Lychee-Docker/issues/190), and I wanted to report that I'm not having any issues with the icons or with menu transparency. In fact, everything looks and works even better in 4.9.0!

**Reply by ildyria** [reply]
https://lycheeorg.github.io/2024-06-29-future-of-lychee/

**Reply by Tiritibambix** [reply]
Thanks a lot for the link, I totally missed that.
I wish you the best of luck on working on this new version and can't wait to try it :)
